webpack.config.js

```diff
@@ -34,14 +36,17 @@ module.exports = {
 		} ),
 		new AppCachePlugin(),
 		new HtmlWebpackPlugin( {
-			title: 'Simplenote',
-			templateContent: require( './index-builder.js' ),
-			inject: false
+			title: 'Simplenote'
 		} ),
 		new webpack.DefinePlugin( {
 			'process.env.NODE_ENV': JSON.stringify(
 				process.env.NODE_ENV || 'development'
-			)
+			),
+			config: JSON.stringify( config() )
```

Comment:
I was hoping to add the ability to read in `git describe` to the config. The `require` in `get-config` would already pick up a `config.json` if one was defined.

I prefer configuration scripts as js instead of `JSON` usually.